README.md

```diff
@@ -48,6 +48,8 @@ graph LR
 
     wf1["WFWX Fire Weather API</br>[Software System]"]
 
+    wso2["WSO2 API Gateway"]
```

Comment:
```suggestion
    ws02["WS02 API Gateway"]
```

Reply:
Vendor says it's "O": https://wso2.com/api-manager/